.github/workflows/release.yml

```diff
@@ -2,6 +2,7 @@ name: Create helm chart issue on release
 on:
   release:
     types: [published]
+  workflow_dispatch:
```

Comment:
is this an intentional change?

Reply:
`workflow_dispatch` is the command to run a workflow on-demand. I was hoping this would give me an option right from this PR to test it, but I'm not seeing it (unless I'm missing it somewhere). So I'm thinking we keep it in so I can try it manually after merging to main to see if it creates an issue properly, and if it doesn't, then we know this workflow still doesn't work (as opposed to waiting for the next release to see that it doesn't work). Open to rejection on that idea though, and just see what happens.